src/arrays/SortingArray.groovy

```diff
@@ -0,0 +1,13 @@
+package arrays
+
+class SortingArray {
+
+    public int[] arrayAsc;
+//    TODO sort reverse array(ArrayDesc)
+//    public int[] arrayDesc;
+
+    SortingArray(int[] arrayValueAsc, int[] arrayValueDesc) {
+        arrayAsc = arrayValueAsc;
+//        arrayDesc = arrayValueDesc;
```

Comment:
get rid of commented code. You should never push commented code.

Also, you still have TWO parameters in your constructor. But when you construct class you passed only ONE argument. Passed arguments should have the same amount and order as your parameters